pkg/cidata/cidata.TEMPLATE.d/boot/04-persistent-data-volume.sh

```diff
@@ -6,6 +6,9 @@ set -eux -o pipefail
 # Restrict the rest of this script to Alpine until it has been tested with other distros
 test -f /etc/alpine-release || exit 0
 
+# Workaround https://github.com/lima-vm/lima/issues/2235
+sed -i 's/^timezone:.*//g' /etc/fstab
```

Comment:
This would be better fixed by adding a blank line in the template before the `timezone:` line:

```diff
--- pkg/cidata/cidata.TEMPLATE.d/user-data
+++ pkg/cidata/cidata.TEMPLATE.d/user-data
@@ -21,6 +21,7 @@ mounts:
 {{- end }}

 {{- if .TimeZone }}
+
 timezone: {{.TimeZone}}
 {{- end }}
```

Reply:
It doesn't seem correct that we have to care about white spaces 😅

Reply:
Anyway, this is being fixed in:
- https://github.com/lima-vm/alpine-lima/pull/124
:thanks:

Reply:
> It doesn't seem correct that we have to care about white spaces 😅

It's because we are parsing YAML with `awk` and not an actual parser...

> Anyway, this is being fixed in:
>
> https://github.com/lima-vm/alpine-lima/pull/124

I don't think it is. It is just not being triggered in Rancher Desktop because it is still using Lima 0.19, which doesn't include the `timezone` code.

We obviously can make the `awk` code better, but adding the blank line will fix things for the existing Alpine image, and looks better in the generated `user-data` anyways.

I noticed it too late that the empty line in the template was being removed by the `{{- … }}` expression and didn't want to create a PR for what I thought was just aesthetics.

Reply:
> We obviously can make the awk code better, 

I will update the script in https://github.com/lima-vm/alpine-lima/pull/124 to handle this better

Reply:
Updated in https://github.com/lima-vm/alpine-lima/pull/126

Reply:
Removed the commit